creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -40,17 +173,16 @@ module.exports = class PrometheusRegistry {
   getMetric(name) {
     return this.registry.getSingleMetric(name)
   }
-}
 
-/**
- * Creates and registers every custom metric, for use throughout application
- */
-const createAllCustomMetrics = function (registry) {
-  for (const { metricType: MetricType, metricConfig } of Object.values(
-    Metrics
-  )) {
-    // Create and register instance of MetricType, with provided metricConfig
-    const metric = new MetricType(metricConfig)
-    registry.registerMetric(metric)
+  // This is to prevent any metric usage before the metrics are properly initialized
+  isInitialized() {
+    return this.initialized
+  }
+
+  // This is to mark the PrometheusRegistry as done with initializing
+  doneInitializing() {
```

Comment:
can we call this function init()? i think that's what other services like libs, blacklist manager, queues etc use